konnectivity-socks5-proxy/main.go

```diff
@@ -74,9 +76,9 @@ func NewStartCommand() *cobra.Command {
 	return cmd
 }
 
-func dialFunc(caCertPath string, clientCertPath string, clientKeyPath string, proxyHostname string, proxyPort int) func(ctx context.Context, network string, addr string) (net.Conn, error) {
+func dialFunc(caCertPath string, clientCertPath string, clientKeyPath string, proxyHostname string, proxyPort int, connectDirectlyToCloudApis bool) func(ctx context.Context, network string, addr string) (net.Conn, error) {
 	return func(ctx context.Context, network string, addr string) (net.Conn, error) {
-		if shouldGoDirect(strings.Split(addr, ":")[0]) {
+		if connectDirectlyToCloudApis && shouldGoDirect(strings.Split(addr, ":")[0]) {
```

Comment:
`shouldGoDirect` should probably be named something like `isCloudAPI`

Reply:
That is a great idea, updated, ptal